packages/ui/react-ui-editor/src/extensions/automerge/defs.ts

```diff
@@ -40,7 +40,7 @@ export interface DocAccessor<T = any> {
 
 // TODO(burdon): Reconcile with echo.
 export const DocAccessor = {
-  getValue: (accessor: DocAccessor) => get(accessor.handle, accessor.path),
+  getValue: (accessor: DocAccessor): string => get(accessor.handle, accessor.path),
```

Comment:
It's not always a string. This would return any value stored at the given path. I'd type this as `never` and then assert that this is a string at the place where this is called.

```suggestion
  getValue: (accessor: DocAccessor): never => get(accessor.handle, accessor.path),
```